includes/Story_Renderer/HTML.php

```diff
@@ -51,6 +51,40 @@ class HTML {
 	 */
 	protected $document;
 
+	/**
+	 * AMP requires the HTML markup to be encoded in UTF-8.
+	 *
+	 * @var string
+	 */
+	const AMP_ENCODING = 'utf-8';
+
+	/**
+	 * Encoding identifier to use for an unknown encoding.
+	 *
+	 * "auto" is recognized by mb_convert_encoding() as a special value.
+	 *
+	 * @var string
+	 */
+	const UNKNOWN_ENCODING = 'auto';
+
+	/**
+	 * Default document type to use.
+	 *
+	 * @var string
+	 */
+	const DEFAULT_DOCTYPE = '<!DOCTYPE html>';
+
+	/**
+	 * Encoding detection order in case we have to guess.
+	 *
+	 * This list of encoding detection order is just a wild guess and might need fine-tuning over time.
+	 * If the charset was not provided explicitly, we can really only guess, as the detection can
+	 * never be 100% accurate and reliable.
+	 *
+	 * @var string
+	 */
+	const ENCODING_DETECTION_ORDER = 'UTF-8, EUC-JP, eucJP-win, JIS, ISO-2022-JP, ISO-8859-15, ISO-8859-1, ASCII';
```

Comment:
Why is this a const, it is only used once?

Reply:
Copied those consts mostly from the AMP plugin. Makes comparing the code easier and might be expanded in the future

Reply:
If that is the case, should you `@link` annotation.